docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -531,6 +531,112 @@ When the batch-consumer mode is enabled, the specific headers of batched message
 
 NOTE: When publish messages, all the above batch headers if exist will be removed from the messages to send.
 
+===== Multi Binders Support
+Step 1. To use multiBinders of EventHubs, we need to configure below properties in application.yml
```

Comment:
"multiBinders" should be "multiple binders". And let's add the reminder here like 
```
This sample takes connection string as example. Credentials of service priciple and MSI are also supported, users can set related properties in each binder's environment settings.
```